refresh/templates/common/Application.swift

```diff
@@ -67,6 +67,10 @@ public func initialize() throws {
 <% if (exampleEndpoints) { -%>
     initializeProductRoutes()
 <% } -%>
+
+    router.get("/health") { request, response, _ in
+        try response.send("{\"status\":\"UP\"}").end()
```

Comment:
We should probably be setting the Content-Type of the response to indicate the JSON data content.